### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by CobbyElsonfx**
##  In this PR I implemented the following:

- [ ] Deleted all the boilerplate from Create React App (text, images, styles).
- [ ] Created a new directory called components.
- [ ] Inside components I created a new Calculator.js file.
- [ ] Implemented the design only using (JSX and CSS)
- [ ] Developed just the UI of the calculator
- [ ] Used functional components

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,123 @@
...
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
```
Comment (Whoistolu) [inline README.md:17]: - [ ] [REQUIRED] Kindly ensure that this project is properly licensed. Right now it is pointing to the project README URL.  you can use this [LICENSE](https://choosealicense.com/licenses/mit/).

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @CobbyElsonfx 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._
- [ ] [REQUIRED] Kindly ensure that all boilerplate that is created by the CRA (texts, images, and styles) is deleted. I can see many in this directory below. Let's remove them.
Screenshot 2023-08-24 at 01 00 14

- [ ] [REQUIRED] As stated by the project requirement, the `Calculator.js` should have at least 1 child component. This means that you should break down the UI and functionality of the calculator into smaller, reusable components. These child components can be defined within in separate files, 


### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

### N/A

Cheers, and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.jsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+          <span>0</span>
+        </div>
+      </div>
+      <div className="buttonsContainer">
+        <button type="button">AC</button>
+        <button type="button">+/-</button>
+        <button type="button">%</button>
+        <button type="button">&divide;</button>
+      </div>
+      <div className="buttonsContainer">
```
Comment (taldr27) [inline src/components/Calculator.jsx]: - [ ] According to the project requirements and the feedback from the previous reviewer, could you please consider incorporating at least one child component into your calculator.js file? You might find it helpful to utilize either the calculatorbutton as a child component or the buttonscontainer.

   image

**Review by taldr27 — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️
Hello @CobbyElsonfx, 

Good job so far! 
You're making good progress on the project, but some issues still need to be addressed before you can move on to the next stage. Keep up the good work! 💪

almost-there

## Highlights 🎉
✅ Professional Readme. 
✅ Linters passing.
✅ Correct use of GitHub Flow.
✅ Nice job creating your functional components.
✅ Incorporates feedback from previous code reviews.

## Required Changes ♻️
_Check the comments under the review._

------
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @taldr27 in your question so I can receive the notification.

If you need any further help or have any questions, feel free to reach out to me on Slack as _**Diego Garcia**_.🎯

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). gif_

**Review by Munsa1 — APPROVED** [review]
Hi @CobbyElsonfx, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Added A Child Component As Requested :heavy_check_mark: 
 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**